programs/uxd/tests/unit_tests/utils/calculate_depositories_mint_collateral_amount.rs

```diff
@@ -5,6 +5,10 @@ mod test_calculate_depositories_mint_collateral_amount {
     use proptest::prelude::*;
     use uxd::utils::calculate_depositories_mint_collateral_amount;
     use uxd::utils::is_within_range_inclusive;
+    use uxd::utils::DepositoryTargetRedeemableAmountAndRedeemableAmountUnderManagement;
+    use uxd::ROUTER_CREDIX_LP_DEPOSITORY_0_INDEX;
+    use uxd::ROUTER_IDENTITY_DEPOSITORY_INDEX;
+    use uxd::ROUTER_MERCURIAL_VAULT_DEPOSITORY_0_INDEX;
 
     fn ui_to_native_amount(ui_amount: u64) -> u64 {
```

Comment:
It's uncanny to use 5 decimals (100k), would use 6 decimals here as it's the main usecase